### PR TITLE
proposal for new streaming (BLOB Handling)

### PR DESCRIPTION
**Pull request by renejeglinsky**
Related to https://github.com/cap-js/docs/pull/590

### CODE REVIEW

File: cds/types.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ The following built-in types are provided:
 
 > <sup>(6)</sup> Mapping to ANSI SQL types are given for comparison. Note though, that you need to have the specification of your target database in mind when considering, for example, length restrictions.
 
+> <sup>(7)</sup> BLOBs hold potentially large amounts of data and handling them can be resource consuming (especially requiring the large amount of `LargeBinary` elements). Please consider using non-large `Binary` data type instead, which are returned as it is.
```
Comment (johannes-vogel) [inline cds/types.md:53]: I think we should rewrite that section and recommend to add the stream annotation as also mentioned in the release notes. Most likely BLOBs are huge media objects. Only if they are small enough, they can fit in regular `Binary` data type.

File: guides/databases.md [hunk 2]
```diff
@@ -345,6 +345,20 @@ db.queryForList("SELECT from sqlite_schema where name like ?", name);
 ```
 </div>
 
+### Reading `LargeBinary` / BLOB {.impl .node}
```
Comment (johannes-vogel) [inline guides/databases.md:348]: lgtm

File: cds/types.md (hunk 1 above)
Comment (renejeglinsky) [reply cds/types.md:53]: Would you rewrite it here? Or remove it here to not annotate LargeBinary with media objects in this generic types section? I definitely like the related release note! That is much clearer to me and better understandable.

Comment (renejeglinsky) [reply cds/types.md:53]: Will remove this here and merge the rest. We can take this as a next step.

**Pull request merged**